CMakeLists.txt

```diff
@@ -100,8 +100,6 @@ endif ()
 # Compiler features.
 #--------------------------------------------------------------------------------------------------
 
-set (CMAKE_CXX_STANDARD 11)
```

Comment:
Revert changes to this file.

Reply:
Upto which commit should I revert? If this line is necessary can I just add it in another commit?

Reply:
You should revert all changes to this file. You need make sure your PR doesn't introduce unnecessary changes, and even more importantly that it doesn't accidentally revert older commits (this is what it's doing right now).